Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -229,6 +229,7 @@ void HackStudioWidget::open_project(const String& root_path)
             LexicalPath::relative_path(absolute_old_path, m_project->root_path()),
             LexicalPath::relative_path(absolute_new_path, m_project->root_path()));
     };
+    Config::write_string("HackStudio", "General", "LastOpenedProjectsPath", root_path);
```

Comment:
`LastOpenedProjectsPath`=>`LastOpenedProjectPath`

Reply:
Can we use `RecentProjects` to get the path of the last opened projects? (Instead of additionally storing it in `LastOpenedProjectsPath`).